frontend/components/RoomCard.vue

```diff
@@ -0,0 +1,196 @@
+<script lang="ts" setup>
+import Rooms from "~/pages/rooms.vue";
+import type { Room } from "~/types/api";
+
+const props = defineProps({
+  room: {
+    type: Object as PropType<Room>,
+    required: true,
+  },
+});
+
+const emit = defineEmits(["delete"]);
+
+const colors = [
+  {
+    // blue
+    background: "#ACBED8",
+    houseLeft: "#D3DCEA",
+    houseMiddle: "#90A6CA",
+    houseRight: "#768CBB",
+  },
+  {
+    // dark green
+    background: "#B7B99D",
+    houseLeft: "#D3D4C2",
+    houseMiddle: "#92956D",
+    houseRight: "#81845E",
+  },
+  {
+    // lavender
+    background: "#D5CCD1",
+    houseLeft: "#E8E2E5",
+    houseMiddle: "#BBABB3",
+    houseRight: "#A4909A",
+  },
+  {
+    // aqua
+    background: "#A5B9BB",
+    houseLeft: "#CAD5D7",
+    houseMiddle: "#799497",
+    houseRight: "#5E797C",
+  },
+  {
+    // rose
+    background: "#EFA8B8",
+    houseLeft: "#F7D4DC",
+    houseMiddle: "#E8849D",
+    houseRight: "#DB587C",
+  },
+  {
+    // purple
+    background: "#EAB7DE",
+    houseLeft: "#F3D7ED",
+    houseMiddle: "#DC8AC7",
+    houseRight: "#CE66B0",
+  },
+  {
+    // yellow
+    background: "#E6BE5E",
+    houseLeft: "#EFDA99",
+    houseMiddle: "#DCA22E",
+    houseRight: "#CD8E25",
+  },
+  {
+    // orange
+    background: "#EDBC84",
+    houseLeft: "#F4D7B4",
+    houseMiddle: "#E49753",
+    houseRight: "#DE7C31",
+  },
+  {
+    // light green
+    background: "#8CDBA4",
+    houseLeft: "#C5EDD1",
+    houseMiddle: "#61C780",
+    houseRight: "#3CAB5F",
+  },
+  {
+    // grey aka finished
+    background: "#B0B0B0", // sc-black-300
+    houseLeft: "#D1D1D1", // sc-black-200
+    houseMiddle: "#888888", // sc-black-400
+    houseRight: "#6D6D6D", // sc-black-500
+  },
+];
+
+const color = computed(() => {
+  if (props.room.completed_at) return colors[9];
+  const base32 = "0123456789ABCDEFGHJKMNPQRSTVWXYZ";
+  // get the last character of the random part of the ulid and decode it
+  const index = base32.indexOf(props.room.id[25]?.toUpperCase() ?? "0");
+  return colors[index % 9];
+});
+
+const lastPlayedAgoMinutes = computed(() => {
+  if (!props.room.last_play_start) return undefined;
+  return Math.floor(
+    (new Date().getTime() - new Date(props.room.last_play_start).getTime()) /
+      1000 /
+      60,
+  );
+});
+</script>
+<template>
+  <div
+    class="group flex h-[294px] w-[362px] flex-col gap-4 rounded-lg bg-sc-black-100 p-7 transition-colors hover:bg-sc-black-200"
+  >
+    <div
+      :style="{ 'background-color': color.background }"
+      class="relative h-[176px] rounded-t-lg border-b-[3px] border-b-black"
+    >
+      <h2 class="relative z-10 p-4 text-2xl font-bold">{{ room.name }}</h2>
+
```

Comment:
bei gestarteten und abgeschlossenen Räumen gehört die Anzahl an Teams in der Card angezeigt, nur bei Startklar ist das eben noch nicht vorhanden, da haben wir uns falsch verstanden, sorry ...